pkg/resource/aws/aws_eip_association_test.go

```diff
@@ -12,6 +13,10 @@ func TestAcc_Aws_EipAssociation(t *testing.T) {
 		TerraformVersion: "0.14.9",
 		Paths:            []string{"./testdata/acc/aws_eip_association"},
 		Args:             []string{"scan", "--filter", "Type=='aws_eip' || Type=='aws_eip_association'", "--tf-provider-version", "3.44.0", "--deep"},
+		RetryDestroy: acceptance.RetryConfig{
+			NumberOfTry: 3,
```

Comment:
Not a big deal but, shouldn't it be `NumberOfTries` or `NumberOfAttempts` instead ?

Reply:
+1 for `Attempts`